src/views/Mutations.vue

```diff
@@ -85,6 +85,7 @@ export function associate (mutations, objects) {
 }
 
 export default {
+  name: 'Mutations',
```

Comment:
The name of the Lumino tabs is defined dynamically with the `name` attribute of the widget wrapped object.

Reply:
Nice.